packages/react/src/components/TextInput/PasswordInput.js

```diff
@@ -6,34 +6,35 @@ import { View16, ViewOff16 } from '@carbon/icons-react';
 import { useNormalizedInputProps } from '../../internal/useNormalizedInputProps';
 import { textInputProps } from './util';
 import { FormContext } from '../FluidForm';
+import * as FeatureFlags from '@carbon/feature-flags';
 
 const { prefix } = settings;
 
 const PasswordInput = React.forwardRef(function PasswordInput(
   {
-    labelText,
     className,
-    disabled,
-    id,
-    placeholder,
-    onChange,
-    onClick,
+    disabled = false,
+    helperText,
     hideLabel,
+    hidePasswordLabel = 'Hide password',
+    id,
     inline,
-    invalid,
+    invalid = false,
     invalidText,
-    helperText,
-    light,
+    labelText,
+    light = false,
+    onChange = () => {},
+    onClick = () => {},
+    onTogglePasswordVisibility,
+    placeholder,
+    size = 'md',
```

Comment:
sorry, last thing. I thought `md` was just for v11? Would we need a ternary here for v10 sizes: small & large?

Reply:
derp, good call. Should I go empty string since there was no default in v10?

Reply:
I guess we can also wait to see if Percy finishes out. Md might already be default since before it was just asking for string.

Reply:
I don't think there are any styles for `md` in v10/v11 since there are no styles for `${prefix}--text-input--md` in `/styles` or `/components` so I guess in a way the default is md size wise, it's just never called out specifically?

Reply:
@abbeyhrt Totally. That's what I was assuming as well. The fact the Percy didn't yell at me for setting `md` is pretty telling, i think. @joshblack can we get some confirmation on these assumptions?